docs/src/main/sphinx/release/release-471.md

```diff
@@ -0,0 +1,67 @@
+# Release 471 (19 Feb 2025)
+
+## General
+
+* Add [](/functions/ai) for textual tasks on data using OpenAI, Anthropic, or
+  other LLMs using Ollama as backend.  ({issue}`25028`)
+* Include split count and total split distribution time in the `EXPLAIN ANALYZE`
+  output. ({issue}`25028`)
+* Add support for JSON logging format to console with `log.console-format=JSON`. ({issue}`25081`)
+* Add further packages for use with Python user-defined functions. ({issue}`25058`)
+* Improve performance for Python user-defined functions. ({issue}`25058`)
+* Improve performance for queries using top N operations. ({issue}`24937`)
+* Prevent failures when using fault-tolerant execution with an exchange manager
```

Comment:
Prevent failures when fault-tolerant execution is configured with an exchange manager that uses Azure storage with workload identity.